client/src/components/PrivateRoute.js

```diff
@@ -0,0 +1,18 @@
+import React, { useContext } from 'react';
+import loginContext from '../context/loginContext';
+import { Redirect, Route } from 'react-router';
+
+const PrivateRoute = ({component: Component, ...rest}) => {
+    const {user} = useContext(loginContext);
+
+    return (
+      <Route {...rest} render={
```

Comment:
YESSS! Very nicely realized the conditional rendering of the route / redirect. :100: